frontend/src/components/ClipboardCopy.vue

```diff
@@ -6,6 +6,9 @@
         <b-button size="sm" text="Button" variant="success" @click="CopyLink">
           {{ $t('gdd_per_link.copy') }}
         </b-button>
+        <b-button variant="outline-info" @click="$emit('show-qr-code-button')">
```

Comment:
make this a filled button, use blue (primary) or info color, dont just make it outlined, this looks not good ;)

Reply:
ok